### PR TITLE
September 29 proposals phase updates

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,3 +91,5 @@ Please see [Contributing to WebAssembly](https://github.com/WebAssembly/design/b
 [numeric-values-in-wat]: https://github.com/WebAssembly/wat-numeric-values
 [instrument-tracing]: https://github.com/WebAssembly/instrument-tracing
 [call-tags]: https://github.com/WebAssembly/call-tags
+[relaxed-dead-code-validation]: https://github.com/WebAssembly/design/issues/1379
```
Comment (conrad-watt) [inline README.md:94]: I don't know on what timescale the proposal repo will be created, but ideally it would be linked here. LGTM otherwise.

Comment (tlively) [reply README.md:94]: @dschuff Can you create a repo?

Comment (chicoxyzzy) [reply README.md:94]: Oh I forgot that repo is a requirement for phase 1 https://github.com/WebAssembly/meetings/blob/master/process/phases.md#1-feature-proposal-community-group

Comment (dschuff) [reply README.md:94]: https://github.com/WebAssembly/relaxed-dead-code-validation

**Review by dschuff — APPROVED** [review]

**Review by aardappel — APPROVED** [review]

**Pull request merged**